src/Components/AdminPage.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { Component } from 'react'
+import GoogleMapReact from 'google-map-react'
+import MapStyle from '../Modules/MapStyle'
+import { Icon, Container } from 'semantic-ui-react'
+import Popup from 'reactjs-popup'
+import EntryPopup from './EntryPopup'
+import axios from 'axios'
+import { connect } from 'react-redux'
+
+class AdminPage extends Component {
+  static defaultProps = {
+    center: {
+      lat: 59.330651,
+      lng: 18.068562
+    },
+    zoom: 11
+  };
+
+  state = {
+    openEntryPopup: false,
+    posts: [],
+    id: '',
+    unpublished: []
+  }
+
+  closeModal = () => {
+    this.setState({ openEntryPopup: false })
+  }
+
+  componentDidMount() {
+    this.axiosGetUnpublished()
+  }
+
+  combineFunctions = () => {
+    this.props.dispatch({ type: 'CHANGE_VISIBILITY' })
+    this.axiosGetUnpublished()
+  }
+
+  async axiosGetUnpublished() {
+    await axios.get('/api/v1/posts').then(response => {
+      this.setState({ posts: response.data })
+    })
+    let unpublished = []
+    await this.state.posts.map(post => {
+      if (post.status === 'pending') {
+        unpublished.push(post)
+      }
+    })
+    this.setState({ unpublished: unpublished })
+  }
+
+  render() {
+    let user = this.props.currentUser.isSignedIn
+    let adminView
+
+    if (user === true && this.props.admin === true) {
+      adminView = (
+        <>
+          <div id='map'
+            onClick={this.props.sidebarVisible ? () => { this.combineFunctions() } : () => { this.axiosGetUnpublished() }}
+          >
+
+          <Icon
+            name='adn'
+            size='huge'
+            color='orange'
+            id='map-icon-admin'
+          />
+
+            <Popup
+              open={this.state.openEntryPopup}
```

Comment:
code formatting

Reply:
if you mean line 71, it's already indented. it's part of the `Popup` properties so it should be like this.